app/lib/common-checks.js

```diff
@@ -75,9 +75,50 @@ const isUserOldWorldRejectWithinTenMonths = (applicationData) => {
   }
 }
 
+const isUserOldWorldReadyToPayWithinTenMonths = (applicationData) => {
+  const dateOfClaim = getValueWhereCondition(applicationData, 'statusId', 9, 'dateOfClaim')
```

Comment:
We should be using the visitDate to measure the 10 month rule not the dateOfClaim

Reply:
thought I asked this question ealier. will replace it